proto/interchain_security/ccv/v1/ccv.proto

```diff
@@ -69,3 +69,33 @@ message UnbondingOpsIndex {
 message MaturedUnbondingOps {
   repeated uint64 ids = 1;
 }
+
+// ConsumerPacketData contains a consumer packet data and a type tag
+message ConsumerPacketData {
```

Comment:
If this protobuf type is defined in `ccv.proto`, I'd prefer that it doesn't have the word "consumer" in it, because that may be confusing if the type is used on both consumer and provider.

In other places such as `throttle.go`, this type is referred to as ThrottledPacketData